packages/core/monaco-editor/package.json

```diff
@@ -33,10 +33,8 @@
     "stylelint": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}'",
     "stylelint:fix": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}' --fix",
     "typecheck": "vue-tsc -p './tsconfig.build.json' --noEmit",
-    "test:component": "BABEL_ENV=cypress cross-env FORCE_COLOR=1 cypress run --component -b chrome --spec './src/**/*.cy.ts' --project '../../../.'",
-    "test:component:open": "BABEL_ENV=cypress cross-env FORCE_COLOR=1 cypress open --component -b chrome --project '../../../.'",
-    "test:unit": "cross-env FORCE_COLOR=1 vitest run",
-    "test:unit:open": "cross-env FORCE_COLOR=1 vitest --ui"
+    "test": "cross-env FORCE_COLOR=1 vitest run",
+    "test:open": "cross-env FORCE_COLOR=1 vitest --ui"
```

Comment:
issue: These scripts are mapped to the CI pipeline and cannot change

You can change what they do, but the name of the script needs to remain as-is per repository standards.